ocaml/xapi/helpers.ml

```diff
@@ -1102,6 +1102,10 @@ let assert_is_valid_cidr kind field cidr =
   if parse_cidr kind cidr = None then
     raise Api_errors.(Server_error (invalid_cidr_address_specified, [field]))
 
+let assert_is_valid_ip_or_cidr kind field address =
```

Comment:
I would call this `is_valid_ip_addr` and not reveal in the name that this could be either IP or CIDR format.